DuckDuckGo/Common/Extensions/StringExtension.swift

```diff
@@ -85,6 +85,24 @@ extension String {
         self.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty
     }
 
+    var cleanStringForBookmarkSearch: String {
```

Comment:
Very picky notes: 
1. In general, functions that get self and return a modified version of self are named like `replacingOccurrences(of:...)`, this could be `replacing....` or `cleaning...`
2. This is a function in a shared framework but is very specific and undocumented, I see 2 options here:
       a.  Use a more generic name and add documentation explaining the input, the expected output, the use cases and examples.
      b. Leave it as it is but move it from the shared extension to a private, bookmarks only, extension